src/test/java/com/karankumar/booksapi/repository/BookSeriesRepositoryTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021  Karan Kumar
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.karankumar.booksapi.repository;
+
+import com.karankumar.booksapi.annotations.DataJpaIntegrationTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DataJpaIntegrationTest
+@DisplayName("BookSeries Repository should")
+public class BookSeriesRepositoryTest {
```

Comment:
Could you please remove this empty test class?

Reply:
Removed.